templates/page.tpl.php

```diff
@@ -89,16 +89,22 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
     <div class="wrapper">
       <section class="footer-top">
         <nav>
-          <?php print render($page['footer']); ?>
+          <?php print render($page['footer_top']); ?>
         </nav>
       </section>
+      <section>
+        <div class="footer-logo"></div>
+        <div class="footer-links">
+          <nav>
+            <?php print render($page['footer']); ?>
+          </nav>
```

Comment:
@rujiali Just to clarify, I think we should do two things:

1. Move the `<nav>` element into a template file for all menu block blocks
1. Provide theme settings to allow the h/v layout of blocks within the footer regions.

That inline with what you're agreeing with?